src/main/target/common_pre.h

```diff
@@ -356,13 +356,11 @@
 #if (TARGET_FLASH_SIZE > 256)
 #define USE_AIRMODE_LPF
 #define USE_CANVAS
-#define USE_DASHBOARD
```

Comment:
But I think the dashboard is useful when I end a test flight and wanna have a basic idea of how many space left in my blackbox flash chip or sdcards

Reply:
Dashboard is not the OSD post-flight stats. It's an externally connected I2C OLED display that can show some information. Extremely rarely used as it requires the user to have a display permanently connected to the quad. Also very few people even know what it is.

Reply:
oh, well, you see, i even don't know what it is. Then just remove it LOL